package-lock.json

```diff
@@ -1,8 +1,8492 @@
 {
   "name": "setup-julia",
-  "version": "1.6.0",
-  "lockfileVersion": 1,
+  "version": "1.6.1",
+  "lockfileVersion": 2,
```

Comment:
> I have absolutely no idea what that's about.

I think you used a newer version and it upgraded the lockfile?

@SleeplessByte Do you know if this is an issue or has any consequences? :D

Reply:
It has consequences, and _Can_ have issues, but it's unlikely a problem for this repo. Go for it.

Reply:
Thank you